website/content/en/blog/2024-03-practical-paths-platform-maturity-kubecon.md

```diff
@@ -0,0 +1,24 @@
+---
+title:  Practical Paths to Platform Maturity: Insights from KubeCon Paris
+slug:   practical-paths-to-platform-maturity
+date:   2024-03-27 12:00:00 +0000
+author: Marsh Gardiner
+categories:
+- Article
+tags:
```

Comment:
I am comfortable with this tagging, though it is edging on community contribution. 

Reason to put `community` is because it is no an official announcement and not formally reviewed (we want this to move quick!)

Reason to not need `community` is because it is more of a sharing of other content (though arguably that is in itself community work) and there were a number of "votes" for this to be something shared more widely.

I don't see this as blocking, but wanted to make it visible as we are still finding our feet on what to tag with what!